receiver/simpleprometheusreceiver/config.go

```diff
@@ -30,6 +31,8 @@ type Config struct {
 	CollectionInterval time.Duration `mapstructure:"collection_interval"`
 	// MetricsPath the path to the metrics endpoint.
 	MetricsPath string `mapstructure:"metrics_path"`
+	// Params the parameters to the metrics endpoint.
```

Comment:
```suggestion
	// Params are the parameters to pass to the metrics endpoint.
```

Reply:
No, this is not how go renders docs. See the rest of the doc - or ScrapeConfig where I lifted this.